src/context/like-video-context.jsx

```diff
@@ -0,0 +1,68 @@
+import axios from "axios";
+import { createContext, useState } from "react";
+
+const LikeVideoContext = createContext();
+
+const LikeVideoProvider = ({ children }) => {
+  const [likeVideos, setLikeVideos] = useState([]);
```

Comment:
suggestion - you can name the state to `likedVideos` for the consistency as the page and handler also have `liked` instead of `like` in their names. One more reason for doing so can be that `likeVideos` sounds like a verb or an action of liking a video which does not suit an array of videos.

Reply:
sure, i will do the suggested changes